src/main/java/me/ryanhamshire/GriefPrevention/events/VisualizationEvent.java

```diff
@@ -63,6 +78,15 @@ public boolean showSubdivides()
         return showSubdivides;
     }
 
+    /**
+     * Check if event was called through shift-inspecting with the inspection tool.
+     * @return True if shift-inspecting
+     */
+    public boolean isNearbyClaims()
```

Comment:
hasMultipleClaims
isVisualizingNearbyClaims

idk, isNearbyClaims is a little too small to be self-descriptive but eh, I can't really think of a better one...

Reply:
I think your second option is also nice. However if we'd look from the context with the class name:
`VisualizationEvent#isNearbyClaims` and `VisualizationEvent#isVisualizingNearbyClaims`

I think they are both fine

Reply:
idk, the former fits better in that context.